src/PKSim.Core/Model/ParameterExtensions.cs

```diff
@@ -105,6 +105,14 @@ public static bool ValueDiffersFromDefault(this IParameter parameter)
          return !ValueComparer.AreValuesEqual(parameter.Value, parameter.DefaultValue.Value, CoreConstants.DOUBLE_RELATIVE_EPSILON);
       }
 
+      //TODO Use ValueOrigin state when implemented. It should be != than PKSim default and probably mereged with method above
+      // This is just a dymmy method for now to satisfy requalification project
```

Comment:
I know that's going to be removed as it's todo but dymmy => dummy if you are in here again.